### PR TITLE
add and use react-tooltip component

### PR DESCRIPTION
**Pull request by gyurka00**
Change styled-tooltip-component to react-tooltip.
(https://github.com/wwayne/react-tooltip)

### CODE REVIEW

**Comment by szerintedmi** [comment]
Deploy preview for *augmint* processing.

Built with commit 92d85ce844c4b124951f6c4f0e99b61a63e9e65f

https://app.netlify.com/sites/augmint/deploys/5b3392b202ed83713dd355a7

File: src/components/toolTip/styles.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+import ReactTooltip from "react-tooltip";
+
+export const StyledToolTip = styled(ReactTooltip)`
+    font-size: 1rem;
+    &.customeTheme {
```
Comment (reyrodrigez) [inline src/components/toolTip/styles.js]: is there a typo in this class name?

File: src/components/toolTip/index.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+                id={id}
+                place="right"
+                effect="solid"
+                type="info"
+                multiline={true}
+                html={html}
+                className="customeTheme"
+                delayHide={200}
+            >
+                {header && <div>{header}</div>}
```
Comment (reyrodrigez) [inline src/components/toolTip/index.js]: could you please use a typographical element like `<h4>`. you might also need to tweak it's style

File: src/components/toolTip/styles.js [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import styled from "styled-components";
+import theme from "styles/theme";
+
+import ReactTooltip from "react-tooltip";
+
+export const StyledToolTip = styled(ReactTooltip)`
+    font-size: 1rem;
```
Comment (reyrodrigez) [inline src/components/toolTip/styles.js]: let's reduce the copy size to `14px` or even to `12px` and add 1.2 line-height to make it more readable.

File: package.json [hunk 4]
```diff
@@ -23,17 +23,16 @@
...
     "redux-logger": "3.0.6",
     "redux-thunk": "2.3.0",
     "redux-watch": "1.1.1",
     "stringifier": "1.3.0",
-    "styled-tools": "0.6.0",
     "styled-components": "3.3.3",
-    "styled-system": "2.3.2",
     "styled-components-grid": "2.0.1",
-    "styled-tooltip-component": "1.3.0",
+    "styled-tools": "0.6.0",
```
Comment (reyrodrigez) [inline package.json]: too soon! :)

File: src/components/toolTip/styles.js (hunk 1 above)
Comment (gyurka00) [reply src/components/toolTip/styles.js]: fixed

File: src/components/toolTip/index.js (hunk 2 above)
Comment (gyurka00) [reply src/components/toolTip/index.js]: fixed

File: src/components/toolTip/styles.js (hunk 3 above)
Comment (gyurka00) [reply src/components/toolTip/styles.js]: yes, I set to 12px with 1.2 line-height

**Pull request merged**